packages/console/src/pages/SignInExperience/PageContent/SignUpAndSignIn/SignUpForm/use-sign-up-password-listeners.ts

```diff
@@ -57,13 +65,16 @@ const useSignUpPasswordListeners = () => {
       }
     );
 
-    if (submitCount) {
+    if (submitCountRef.current > 0) {
       // Wait for the form re-render before validating the new data.
       setTimeout(() => {
         void trigger('signIn.methods');
       }, 0);
     }
-  }, [getValues, setValue, signUpPassword, submitCount, trigger]);
+  }, [getValues, setValue, signUpPassword, trigger]);
 };
 
 export default useSignUpPasswordListeners;
+function trigger(arg0: string) {
```

Comment:
What is this

Reply:
looks like a dummy vscode auto complete generated code. Removed.